src/api/ServerMetadata.ts

```diff
@@ -51,6 +51,11 @@ export class ServerMetadata {
     return this.version.ge('15.0.2');
   }
 
+  /** Is it safe to use JSON for most operations? */
+  public useJson() {
+    return this.version.ge('19.0.0');
```

Comment:
We need to handle Meridian versions here too. This current statement will match 2015.1.0, which was based on versions before 19.0.0.